spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -357,12 +361,29 @@
     }]
   end
 
+  let(:multiple_rights) { ["https://opensource.org/licenses/MIT", "http://creativecommons.org/publicdomain/zero/1.0/"] }
+  let(:expected_multiple_rights) do
+    [{
+      "rights": "MIT License",
+      "scheme": "spdx",
+      "identifier": "MIT",
+      "url": mit_license_url
+    },
+    {
+      "rights": "Creative Commons Zero v1.0 Universal",
+      "scheme": "spdx",
+      "identifier": "CC0-1.0",
+      "url": creative_commons_zero_url
+    }]
+  end
```

Comment:
Are these the expected exported rights? If so, they will be changed for https://inveniordm.docs.cern.ch/reference/metadata/#rights-0-n correct?

Reply:
Ah, yes. This will need to be updated once #878 is merged.